fdbserver/workloads/MetaclusterManagementConcurrencyWorkload.actor.cpp

```diff
@@ -281,12 +293,21 @@ struct MetaclusterManagementConcurrencyWorkload : TestWorkload {
 
 		state Optional<int64_t> newNumTenantGroups;
 		state Optional<ClusterConnectionString> connectionString;
+		state Optional<metacluster::AutoTenantAssignment> autoTenantAssignment;
 		if (deterministicRandom()->coinflip()) {
 			newNumTenantGroups = deterministicRandom()->randomInt(0, 4);
 		}
 		if (deterministicRandom()->coinflip()) {
 			connectionString = dataDb.getReference()->getConnectionRecord()->getConnectionString();
 		}
+		{
+			int rnd = deterministicRandom()->randomInt(0, 5);
```

Comment:
Similar comment about using probabilities. My suggestion would be to use a `coinflip` to determine whether this is set and a second `coinflip` to determine its value.

Reply:
Done